centreon/www/include/common/listing/listing.css

```
@@ -0,0 +1,1018 @@
+    font-size: 14px;
+}
+
+.cl-listing-table tbody {
+    transition: opacity 0.2s ease;
+}
+
+.cl-listing-table tbody.cl-fade-in {
+    animation: clFadeIn 0.25s ease;
+}
+
+@keyframes clFadeIn {
```

Stylelint (17.5.0)

[error] Expected keyframe name "clFadeIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)